src/features/CareViewPatients/utils/CareViewPatientsUtils.js

```diff
@@ -3,12 +3,28 @@ import {
   BOOKMARK_PATIENT_BASE_URL,
   GET_PATIENT_LIST_URL,
   GET_SEARCH_PATIENT_LIST_URL,
+  GET_MY_PATIENT_LIST_URL,
 } from "../../../constants";
 
-export const fetchPatientsList = async (wardId, offset, limit = 3) => {
-  return await axios.get(GET_PATIENT_LIST_URL.replace("{wardId}", wardId), {
-    params: { offset, limit },
-  });
+export const fetchPatientsList = async (
+  wardId,
+  offset,
+  limit = 3,
+  headerSelected,
+  providerUuid
+) => {
+  if (headerSelected === "TOTAL_PATIENTS") {
```

Comment:
Can we push these string literals into the constants folder? I see it is being used in multiple places
`MY_PATIENTS` and `TOTAL_PATIENTS`

Reply:
Sure I will do. Thanks